substrate/primitives/io/src/lib.rs

```diff
@@ -1219,18 +1219,31 @@ pub trait Crypto {
 			.expect("`bls381_generate` failed")
 	}
 
-	/// Generate an `(ecdsa,bls12-381)` key for the given key type using an optional `seed` and
+	/// Generate a 'bls12-381' Proof Of Possession for the corresponding public key.
+	///
+	/// Returns the Proof Of Possession as an option of the ['bls381::Signature'] type
+	/// or 'None' if an error occurs.
+	#[cfg(feature = "bls-experimental")]
+	fn bls381_generate_pop(
```

Comment:
I imagine that a new host call requires an RFC proposal?

Reply:
As this basically adds a new HF, I guess it  requires a dedicated RFC. I.e. an RFC dedicated to the introduction of the bls+ecdsa crypto scheme

Reply:
Yeah let push PoP to production before dealing with bls specific stuff.